config/mercure.yaml

```
@@ -20,7 +20,8 @@ services:
         arguments:
             $tokenProvider: '@Pimcore\Bundle\StudioBackendBundle\Mercure\Service\ClientTokenService'
             $cookieLifetime: '%pimcore_studio_backend.mercure_settings.cookie_lifetime%'
-            $cookieSameSite: '%pimcore_studio_backend.mercure_settings.cookie_same_site%'
+            $jwt_cookie_strictness: '%pimcore_studio_backend.mercure_settings.jwt_cookie_strictness%'
```

```suggestion
            $jwtCookieStrictness: '%pimcore_studio_backend.mercure_settings.jwt_cookie_strictness%'
```
to keep camelCase as the others

---

maybe even just `$cookieStrictness`, it's always JWT in this context